extensions/wikia/NjordPrototype/css/Njord.scss

```diff
@@ -261,14 +261,21 @@
 	}
 
 	.title-wrap {
-		margin: 30px 330px 20px 30px;
+		margin: 0;
```

Comment:
sorry this line shouldn't be there it was a leftover that I forgot to remove. My mistake, and good catch :+1: